README.md

```diff
@@ -0,0 +1,136 @@
+
+# Usage
+
+Build and install AS components:
+
+```shell
+git clone https://github.com/confidential-containers/attestation-service
+cd attestation-service
+make && make install
+```
+
+`grpc-as` and `grpc-as-ctl` will be installed into `/usr/local/bin`.
+
+# Architecture
+
+The main architecture of the Attestation Service is shown in the figure below:
+```
+┌──────────────────────┐      Evidence        ┌─────────────────────────────┐
+│                      ├──────────────────────>     Attestation Service     │
+│Verification Demander │                      │                             │
+│    (Such as KBS)     <──────────────────────┤┌──────────┐┌───────────────┐│
+│                      │  Attestation Result  ││  Policy  ││Reference Value││
+└──────────────────────┘                      ││  Engine  ││    Povider    ││
+                                              │└──────────┘└───────────────┘│
+                                              │┌───────────────────────────┐│
+                                              ││     Verifier Drivers      ││
+                                              │└───────────────────────────┘│
+                                              └─────────────────────────────┘
+```
+
+### Evidence format:
+
+The format of Evidence should be consistent with [KBS protocol](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md).
+
+```json
+{
+    "tee-pubkey": $pubkey,
+    "tee-evidence": {}
+}
+```
+
+- `tee-pubkey`: A JWK-formatted public key, generated by the KBC running in the HW-TEE.
+You can see more details of this key's information and effect in the [KBS protocol](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md) description document
+
+- `tee-evidence`: The attestation evidence generated by the HW-TEE platform software and hardware in the AA's execution environment.
+The tee-evidence formats depend on the TEE and are typically defined by the each TEE verifier driver of AS.
+
+**Note**: Verification Demander needs to specify the TEE type and pass `nonce` to Attestation-Service together with Evidence,
+Hash of `nonce` and `tee-pubkey` should be embedded in report/quote in `tee-evidence`, so they can be signed by HW-TEE.
+This mechanism ensures the freshness of Evidence and the authenticity of `tee-pubkey`.
+
+### Attestation result format:
+
+```json
+{
+    "tee": $tee_type,
+    "allow": true,
+    "output": {
+        "verifier_output": $verifier_output,
+        "policy_engine_output": $opa_output
+    },
+    "tcb_status": $tcb_status_claims
+}
+```
+
+The `output` is the result log of verifier driver and policy engine.
+When attestation verification result is `false`, you can find the reject reason logs in `output`. 
+When attestation verification result is `true`, `verifier_ouput` will be let blank and `policy_engine_output` can output detailed customized content, depending on how users write their own policy.
```

Comment:
Suggestion:

```

* `allow`: The verification results. `true` if verification succeeded, `false` otherwise.
* `output`: When verification fails, `output.verifier` describes the failure reasons. When verification succeeds, `output.policy_engine` may provide additional policy verification information, depending on the policy configuration. 

```